isv/src/constants.py

```diff
@@ -1,22 +1,52 @@
-LOSS_ATTRIBUTES = GAIN_ATTRIBUTES = [
+# LOSS_ATTRIBUTES = GAIN_ATTRIBUTES = [
```

Comment:
nezakomentovaval by som to ale rovno zmazal, mame git v ktorom sa stare verzie daju pozriet

Reply:
zmazane, dakujem :)